powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/replay/ReplayPersistenceService.java

```diff
@@ -70,7 +70,7 @@ public boolean uniqueValueExists(final String uniqueValue) {
      * @return Whether unique value was added successfully.
      */
     public boolean persistUniqueValue(final UniqueValueType type, final String uniqueValue) {
-        final Instant expiration = Instant.now().plus(config.getRequestExpirationInMilliseconds(), ChronoUnit.MILLIS);
+        final Instant expiration = Instant.now().plus(config.getRequestExpiration().toMillis(), ChronoUnit.MILLIS);
```

Comment:
Doesn't it work in a simpler form?
```suggestion
        final Instant expiration = Instant.now().plus(config.getRequestExpiration());
```